src/cvl/custom_validation/sonic_sflow.go

```diff
@@ -0,0 +1,31 @@
+package custom_validation
+
+import (
+	"net"
+	util "cvl/internal/util"
+ )
+
+
+//Validate ip address of sflow collector ip 
+func (t *CustomValidation) ValidateIp(vc *CustValidationCtxt) CVLErrorInfo {
+	ip := net.ParseIP(vc.YNodeVal)
+	if ip == nil {
```

Comment:
Since the collector_ip is a mandatory leaf and enforced by CVL, the code to check the nil is redundant.

Reply:
But the ip == null check is done after ParseIP().  I guess this check fine, otherwise if for some reason ip is null we will have crash due to the code below?

Reply:
Yes. ParseIp() returns null if the conversion from IP (string) to IP (Type IP)  fails.